index.html

```diff
@@ -947,620 +974,1301 @@
+                      <img
+                        src="./assets/trump-kwame.JPG"
+                        alt="trump lowers sentence for kwame"
+                        class="img-fluid Us-small-img"
+                      />
+                    </div>
+                    <div class="col-lg-8 Us-small-text pr-0 pt-1">
+                      <strong
+                        >'Tiger King' Zoo Owners Ordered To Hand Over Big Cats
+                        to Government</strong
+                      >
+                    </div>
+                  </div>
                 </div>
-                <div class="text">
-                  "Unity starts with accountability," Warren said Sunday during
-                  an interview with CNN's Abby Phillip.
+                <div class="container pr-0 pt-1">
+                  <div class="row border-bottom pb-2">
+                    <div class="col-lg-4 px-0 pt-1">
+                      <img
+                        src="./assets/trump-kwame.JPG"
+                        alt="trump lowers sentence for kwame"
+                        class="img-fluid Us-small-img"
+                      />
+                    </div>
+                    <div class="col-lg-8 Us-small-text pr-0 pt-1">
+                      <strong
+                        >'Tiger King' Zoo Owners Ordered To Hand Over Big Cats
+                        to Government</strong
+                      >
+                    </div>
+                  </div>
+                </div>
+                <div class="container pr-0 pt-1">
+                  <div class="row pb-2">
+                    <div class="col-lg-4 px-0 pt-1">
+                      <img
+                        src="./assets/trump-kwame.JPG"
+                        alt="trump lowers sentence for kwame"
+                        class="img-fluid Us-small-img"
+                      />
+                    </div>
+                    <div class="col-lg-8 Us-small-text pr-0 pt-1">
+                      <strong
+                        >'Tiger King' Zoo Owners Ordered To Hand Over Big Cats
+                        to Government</strong
+                      >
+                    </div>
+                  </div>
                 </div>
               </div>
             </div>
-            <div class="nstorty">
-              <img src=./assets/cindy-mccain-doug-ducey-funeral.jpg alt="biden"
-              class="images-float-left">
-              <div class="heading_text">
-                <div class="bheading">
-                  Warren Calls for Unity 'Against Insurrection' as Trump
-                  Impeachment Delayed
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-lg-12 px-0 subscribe-container border-top pt-4">
+            <div class="container px-0">
+              <div class="row">
+                <div class="col-lg-9 pr-4">
+                  <p class="subscribe-title">SUBSCRIBE</p>

```

Comment:
- [ ] This should be a header tag(h2, h3, h34, etc), that guarantees the code is more semantic and accessible to screen readers.